modules/RTC/RTCUtils.js

```diff
@@ -210,11 +210,44 @@ function RTCUtils(RTCService)
                 return this.audioTracks;
             };
         }
-    }
-    else
-    {
+    } else if (window.rtcninja) {
+      console.log('This appears to be an rtcninja platform');
```

Comment:
This `console.log` should be wrapped in a try catch to handle browsers that don't have it. (see line 250)